storybook/src/components/tabs/tabs.css-stories.ts

```diff
@@ -23,4 +23,4 @@ const { Default, Inactief } = tabsStories({
   },
 });
 
-export { Default, Inactief };
+export { AsAnchors, AsAnchorsDisabled };
```

Comment:
Zullen we hier een inline comment plaatsen at we `AsButtons` en `AsButtonsDisabled` niet ondersteunen in de HTML/CSS implementatie?

Reply:
Gedaan